src/component/axis/AxisBuilder.ts

```diff
@@ -435,7 +435,7 @@ const builders: Record<'axisLine' | 'axisTickLabel' | 'axisName', AxisElementsBu
                 text: name,
                 font: textFont,
                 overflow: 'truncate',
-                width: maxWidth,
+                width: maxWidth || +textStyleModel.get('width') || undefined,
```

Comment:
What if a user set the width to be 0? Is this valid? (It seems to be valid to me.) If it is valid, would `|| undefined` make it be `undefined` instead of 0? Is this intended or what does `|| undefined` do?

Reply:
这个是运行测试看到不设置width坐标轴名称会消失，所以加了下，不知道如何设置好。